### PR TITLE
Updating the index page with different links

### PR DESCRIPTION
**Pull request by gcrev93**

### CODE REVIEW

**Comment by PRMerger13** [comment]
@gcrev93 : Thanks for your contribution! The author(s) have been notified to review your proposed change.

**Comment by Mikejo5000** [comment]
@gcrev93 Changes look good. However, the YML file doesn't support conceptual versioning (VS 2022 | VS 2019 | VS 2017), and until Dev17 GA most customers will come to VS 2019 docs and want help with VS 2019. I will edit these with some version-specific information to support both doc views.

**Comment by PRMerger10** [comment]
@Mikejo5000 : Thanks for your contribution! The author(s) have been notified to review your proposed change.

**Comment by PRMerger20** [comment]
@Mikejo5000 : Thanks for your contribution! The author(s) have been notified to review your proposed change.

**Comment by Mikejo5000** [comment]
#sign-off

File: docs/javascript/index.yml [hunk 1]
```diff
@@ -37,14 +37,25 @@ landingContent:
   # Card (optional)
   - title: Create JavaScript and TypeScript apps
     linkLists:
+      - linkListType: quickstart
+          - text: Create an Angular App
```
Comment (Jak-MS) [inline docs/javascript/index.yml:41]:
```suggestion
      - linkListType: quickstart
         links:
          - text: Create an Angular App
```

we'll need to move this to the private repo so we can check the build on staging before we push to master.... but I believe the above line should be added.

**Comment by Jak-MS** [comment]
@gcrev93 @Mikejo5000 - changes to an index.yml can be tricky... so we'll need to take this over to the private repo so we can see it on staging. 

@Mikejo5000  can you open a pr in MicrosoftDocs/visualstudio-docs-pr ?  Alternatively we can move the commits in this PR for you.

**Comment by Mikejo5000** [comment]
@Jak-MS If you can move the commits, that would be great! Thanks!

**Comment by PRMerger14** [comment]
@Jak-MS : Thanks for your contribution! The author(s) have been notified to review your proposed change.

**Comment by jborsecnik** [comment]
@Mikejo5000, I merged this by accident (not realizing it was expected to be moved to the private repo). I will follow up and make sure the page is ok when it's live.  -jborsecnik

**Pull request merged**